.eslintrc.json

```diff
@@ -77,6 +84,7 @@
     "prefer-destructuring": "off",
     "no-multi-assign": "off",
     "no-param-reassign": "off",
+    "lines-between-class-members": [2, "always", { "exceptAfterSingleLine": true }],
```

Comment:
We can skip this rule, no?

Reply:
I am not activating the rule I am changing its configuration. The existing rule has "exceptAfterSingleLine" set to false, which means that eslint would require an empty line between class members like in this case: https://github.com/jhipster/generator-jhipster/blob/c81c32e79edf6ee1bcdace4fe632ad88c9ae3e22/jdl/models/jdl-relationship.ts#L27-L36

The rule didn't matter before because there were no single line class members but now there are some because TypeScript excepts that every property on a class is defined and so I had to add these.

IMO empty lines between single-line class members make the class needlessly large.